### ISSUE TITLE
"Total Works" count inconsistency

### ISSUE THREAD

**Issue by alishaevn** [samvera-labs/bulkrax#351]
# Summary
Sometimes the "Total Works Count" column on the importer/index page and the "Total Works" label on the importer/show page is incorrect.

The number is being set by this code
``` ruby
def total
  if importer?
    return @total if @total&.positive?

    # windows encoded
    @total = `grep -c ^M #{real_import_file_path}`.to_i - 1

    # unix encoded
    @total = `grep -vc ^$ #{real_import_file_path}`.to_i - 1 if @total < 1
  else
    ...
  end
  return @total
end
```

# Example
I uploaded a csv that has 2 rows (not including the header), but it returned as 5
running the above in the console returns:
``` bash
irb(main):001:0> real_import_file_path = '.../abcdefg.csv'
=> ".../abcdefg.csv"

irb(main):002:0> @total = `grep -c ^M #{real_import_file_path}`.to_i - 1
=> -1

irb(main):003:0> @total = `grep -vc ^$ #{real_import_file_path}`.to_i - 1 if @total < 1
=> 5
```
![image](https://user-images.githubusercontent.com/29032869/133330380-8824d725-ca96-47c5-8324-ba7584370235.png)

# Resources
- https://stackoverflow.com/questions/2650517/count-the-number-of-lines-in-a-file-without-reading-entire-file-into-memory
the above link is in the code as reference.

**Comment by alishaevn** [comment]
I paired with steven on this and our results were:
> Looked at this with Alisha.  It looks like this will not be possible unless the data in each row is scrubbed of new lines.  When the data contains quoted columns with new lines it is impossible to tell which new line is the right new line for the row ( unless you want to write a csv parser with sed :dizzy_face: ) . So, something that understands CSVs and all the weird ways that they can be put together is needed to get the right line count eg: ruby’s CSV library.

**Comment by alishaevn** [comment]
closing as this was resolved in #352
